doc/sphinx/source/references.bib

```diff
@@ -65,6 +65,19 @@ @book{hughes2012finite
   publisher={Courier Corporation}
 }
 
+@article{libceed-joss-paper,
+  author       = {{Jed Brown and Ahmad Abdelfattah and Valeria Barra and Natalie Beams and Jean Sylvain Camier and Veselin Dobrev and Yohann Dudouit and Leila Ghaffari and Tzanio Kolev and David Medina and Will Pazner and Thilina Ratnayaka and Jeremy Thompson and Stan Tomov}},
+  title        = {{libCEED: Fast algebra for high-order element-based discretizations}},
+  journal      = {{Journal of Open Source Software}},
+  year         = {2021},
+  publisher    = {The Open Journal},
+  volume       = {6},
+  number       = {63},
+  pages        = {2945},
+  doi          = {10.21105/joss.02945},
+  url          = {https //doi.org/10.21105/joss.02945}
```

Comment:
I drop the `url` field when it's repetitive of the `doi` because it tends to get mangled in bib formats that are capable of displaying `url`.

Reply:
Done. It is present in the other two refs we have in the README though. If we want to drop all of them to be consistent, I'll do it

Reply:
Yeah, let's be consistent on that.

Reply:
Done